_posts/2022-09-20-Project-Sabine-Low-cost-DIY-underwater-modem-using-COTS-components-and-Unet-audio.md

```diff
@@ -0,0 +1,326 @@
+To transmit a message, the user can use the various interfaces provided by Unet audio such as the web interface, shell, or various APIs in a variety of computer languages such as Java, Groovy, Python, C, Julia, Javascript, etc. The same interfaces can be used to configure various communication parameters to achieve the best performance in a given environment.
+
+> NOTE: UnetStack refers to a collection of technologies for extending networks underwater of which Unet audio is one of the components. An interested user may refer to the [Unet handbook](https://unetstack.net/handbook/unet-handbook_unet_basics.html) to gain a deeper understanding of it.
+
+The user-provided data is then converted to passband signals in Unet audio. This includes transport and network layer processing (as required), fragmentation and reassembly, forward error correction (FEC), modulation, and demodulation depending on the communication scheme used. The passband signal (digital) is then converted to an analog signal using a sound card and passed through a power amplifier before being fed to the transducer/underwater speaker for transmission through the medium.
+
+On the receive side, the signal is received using a hydrophone/underwater mic, converted to a digital signal using a sound card's mic in port, and digitally processed to detect and decode the data from the signal using Unet audio. On successful decoding, the data is passed back to the user. If the decoding fails, an appropriate notification is provided to the user.
+
+So the main hardware components of an SDOAM will be an SBC to run Unet audio, a sound card (with speaker out and mic in), audio amplifiers, and speaker and mic. The following sections list the hardware chain for both the transmit and receive chains.
+
+### Transmit chain
+
+Connect the speaker output from the sound card (either a discrete or USB sound card connected to an SBC) to the amplifier module input. The amplifier output is then connected to the underwater speaker. Depending on the amplifier used, it can be powered using the USB output of the RPi or a 12V battery.
+
+<p align="center"><img width="70%" src="../assets/img/sabine/txc.png"/></p>
+<p align="center"><em>Transmit chain</em></p>
+
+### Receive chain
+
+For the receiver side, the underwater microphone is connected directly to the mic in of the soundcard of the SBC.
+
+<p align="center"><img width="70%" src="../assets/img/sabine/rxc.png"/></p>
+<p align="center"><em>Receive chain</em></p>
+
+
+
+## Hardware Selection
+
+
+### Digital Hardware
+
+One of the most important hardware components of an SDOAM is the digital hardware that runs Unet audio. We have selected a Raspberry Pi (3b+) due to its popularity, low cost, ease of availability, and community support to be the digital hardware to run Unet audio. This can easily be swapped with any similar SBCs that have an arm processor running Linux on it. A USB sound card connected to the Raspberry Pi is used as the data acquisition system.
```

Comment:
```suggestion
One of the most important hardware components of an SDOAM is the digital hardware that runs Unet audio. We have selected a Raspberry Pi (3b+) due to its popularity, low cost, ease of availability, and community support to be the digital hardware to run Unet audio. This can easily be swapped with any similar SBCs that have an arm processor running Linux on it. A [USB sound card](https://www.aliexpress.com/item/4000598865391.html) connected to the Raspberry Pi is used as the data acquisition system.
```